docs/operating-scylla/nodetool-commands/compact.rst

```diff
@@ -15,27 +15,11 @@ Syntax
 -------
 .. code-block:: console
 
-   nodetool [options] compact [--partition <partition_key>] [<keyspace> [<cfnames>]...]
+   nodetool [options] compact [<keyspace> [<cfnames>]...]
 
 Options
 --------
 
-* ``-h <host>`` or  ``--host <host>`` - Node hostname or IP address.
-
-* ``-p <port>`` or ``--port <port>`` - Remote JMX agent port number.
-
-* ``--partition <partition_key>`` - String representation of the partition key.
-
-* ``-pp`` or ``--print-port`` - Operate in 4.0 mode with hosts disambiguated by port number.
-
-* ``-pw <password>`` or ``--password <password>`` - Remote JMX agent password.
-
-* ``-pwf <passwordFilePath>`` or ``--password-file <passwordFilePath>`` - Path to the JMX password file.
-
-* ``-u <username>`` or ``--username <username>`` - Remote JMX agent username.
-
-* ``--`` - Separates command-line options from the list of argument(useful when an argument might be mistaken for a command-line option).
-
 The following options are NOT supported:
```

Comment:
Perhaps not related to this patch, but I don't understand this phrasing. After all, the "--dog" option also is not supported. Perhaps this sentence was trying to say that Cassandra supports those but Scylla doesn't? If so, can we say a few words why? Is this something we don't **want** to support, or something we do want to support but just didn't get around to?

Reply:
These are options that `nodetool` itself has, but the backend (Scylla+JMX) doesn't support.